packages/field/src/useField.ts

```diff
@@ -8,7 +8,16 @@
 import { useState } from 'react';
 import { generateId } from '@zendeskgarden/container-utilities';
 
-export function useField(idPrefix) {
+export interface IUseFieldPropGetters {
+  getHintProps: <T>(options?: T & React.HTMLProps<any>) => React.HTMLProps<any>;
+  getLabelProps: <T>(options?: T & React.HTMLProps<any>) => React.HTMLProps<any>;
+  getInputProps: <T>(
+    options?: T & React.HTMLProps<any>,
+    isDescribedOptions?: { isDescribed: boolean }
+  ) => any;
```

Comment:
Since `getInputProps` doesn't receive any required options you can have the same type definition as above:

```tsx
getInputProps: <T>(options?: T, isDescribedOptions?: ...) => T & React.HTMLProps<any>;
```